jupyterbook/_config.yml

```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
```

Comment:
@MathewBiddle I have a preference for the classic style notebook but I am old. What is your opinion on this?

Reply:
@ocefpaf @lohithmunakala, can you point me to a jupyterlab example?

Reply:
This feature is specifically when we have a JupyterHub to run these notebooks just like running them on Colab/Binder. Other than that there isn't any change in the actual appearance of the Jupyter{Book}.

However, I've added examples of classic and JupyterLab below, in case we have JupyterHub, we can use either of these to run the notebooks.
#### Below is a JupyterLab Notebook
![Screenshot from 2021-06-22 00-25-32](https://user-images.githubusercontent.com/56470584/122813570-b8d6c900-d2f0-11eb-95a2-2c430c0ecbb5.png)
#### Below is a Classic Jupyter Notebook
![Screenshot from 2021-06-22 00-25-39](https://user-images.githubusercontent.com/56470584/122813575-ba07f600-d2f0-11eb-89fa-826cd4ea35f8.png)

Reply:
I kind of prefer the JupyterLab view. Reminds me of how I have Pycharm configured :P But, I don't have strong attachments to either.